.eslintrc.json

```diff
@@ -27,6 +27,7 @@
   },
   "rules": {
     "@typescript-eslint/ban-types": "off",
+    "@typescript-eslint/no-explicit-any": "warn",
```

Comment:
@Hyperkid123 I would prefer to leave this as an error and ignore individual lines we see fit. Otherwise much of the value of types is bypassed/ignored unless we check warnings.